app/controllers/courses_controller.rb

```diff
@@ -127,6 +127,11 @@ def alerts
     @alerts = current_user&.admin? ? @course.alerts : @course.public_alerts
   end
 
+  def course_stats
```

Comment:
I think it's going to be easier to include these new stats as part of `show.json`, rather than having a separate route and separate request for them.

Also, I think we should finish and merge and deploy the database and update cycle changes (with or without sending the stats to the frontend in `show.json`) before adding any frontend changes to the PR.

What's left from your perspective before this would be ready to merge?

Reply:
I reset the last commit and I can make that show.json fast. And the other stuff... Maybe to fix CVS to be made from table